tests/test_ncbi.py

```diff
@@ -0,0 +1,54 @@
+    actual_response = list(_get_eutil_records(eutil,id))
+    assert isinstance(_get_eutil_records(eutil, id), types.GeneratorType)
+    assert actual_response == expected_response
```

Comment:
Avoid calling the function twice if you don't have to

```suggestion
    actual_response = _get_eutil_records(eutil,id)
    assert isinstance(actual_response, types.GeneratorType)
    assert list(actual_response) == expected_response
```